src/debug/debugUtils.ts

```diff
@@ -79,3 +79,16 @@ export async function getCommandsOfProcesses(kubectl: Kubectl, pod: string, podN
 
     return commandLines;
 }
+
+const isBashOnContainer = async (kubectl: Kubectl, podName: string, podNamespace: string | undefined, containerName: string | undefined): Promise<boolean> => {

```

Comment:
`async function` rather than `const`?  Some older code uses the "const of function type" design and I'm not sure why, but we are using plain ol' functions in most places.